packages/eslint-plugin/src/rules/shorthand-property-sorting/__tests__/rule.test.ts

```diff
@@ -0,0 +1,173 @@
+import { outdent } from 'outdent';
+
+import { typeScriptTester as tester } from '../../../test-utils';
+import { shorthandFirst } from '../index';
+
+const packages = ['css', 'styled', 'cssMap', 'xcss'];
+
+tester.run('shorthand-property-sorting', shorthandFirst, {
+  valid: [
+    ...packages.map((pkg) => ({
```

Comment:
looks a bit messy...

i suggest generating all the combinations in advance

```tsx
const packages = ['css', 'styled', 'cssMap', 'xcss'] as const;
const imports = ['@atlaskit/css', '@compiled/react'] as const;

const packages_and_imports = packages.flatMap((pkg) => imports.map((imp) => [pkg, imp]));

tester.run('shorthand-property-sorting', shorthandFirst, {
  valid: [
    ...packages_and_imports.map(([pkg, imp]) => ({
      name: `correct property ordering (${pkg})`,
      code: outdent`
      import {${pkg}} from '${imp}';
      // and so on...
```

`@atlaskit/css` is a planned wrapper for `@compiled/react` that we'll need to treat as equivalent to `@compiled/react`

Reply:
in AFM I only ever saw `xcss` imported from `@atlaskit/primitives`. Can we treat it like the others like you suggested?